multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -23,6 +23,13 @@ metadata:
   name: antrea-mc-controller-role
   namespace: antrea-multicluster
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - watch
+  - list
```

Comment:
There will be an error when I used the "list" verb. Maybe somewhere else needs to list the ServiceAccount. I'll check it. Thanks

Reply:
I changed the verbs from "list, get, watch" to "get, watch" and then got an error. Maybe "list" verb is necessary. I will figure out which part needs to list the ServiceAccount.

Reply:
Ok. Let us make it clear. I feel list should not be needed.

Reply:
List verbs may be needed. I check the code. The client of the webhook is created by the mgr. And it's a cached-client. So that the client will list & watch the resources when getting resources.  I changed the client to a non-cached client and it doest not need the watch and list permssion.